packages/internal/src/paths.ts

```diff
@@ -112,6 +111,8 @@ export const resolveFile = (
  */
 export const getPaths = (BASE_DIR: string = getBaseDir()): Paths => {
   const routes = resolveFile(path.join(BASE_DIR, PATH_WEB_ROUTES)) as string
+  const { schemaPath } = getConfig(getConfigPath()).api
```

Comment:
```suggestion
  const { schemaPath } = getConfig().api
```

I think `getConfig()` has a default value that allows it to determine it's own path.

Reply:
Fixed in https://github.com/redwoodjs/redwood/pull/1424/commits/6bbda69f92adcaf6b733e274536366fa9fe0f7ba.